### DISCUSSION TITLE
UserWarning: Failed to initialize NumPy

### DISCUSSION THREAD

**Discussion by MastracciN** [coqui-ai/TTS#1662] [General Q&A]
Hey everyone, I'm trying to install and run the project but I am met with this error. Most solutions I've read online say to upgrade but the project requires 1.21.6 which is the current version I have installed. Any thoughts would be appreciated 

![image](https://user-images.githubusercontent.com/38020285/174134827-eccfb12a-6892-437f-9b2c-60d8c8eb3bd2.png)

**Comment by p0p4k** [comment]
Try:
pip install --upgrade numpy
And run the "tts" command.
Report back, thanks.

**Reply by Lartza** [reply]
Seemingly works (did only cursory tts-server which didn't run before, and tts --text test) if numba is also upgraded, of course with tts warning about
```
tts 0.7.1 requires numba==0.55.1, but you have numba 0.55.2 which is incompatible.
tts 0.7.1 requires numpy==1.21.6, but you have numpy 1.22.4 which is incompatible.
```

**Reply by p0p4k** [reply]
I think updating both numba and numpy fixes the problem. 
The installed Numpy works on its own, but it is not the correct version for pytorch. So update that and then update numba, cause new version numpy is not supported by old numba.
